examples/hyrax.yaml

```diff
@@ -7,597 +9,600 @@
+    display_label: "Keyword"
+    definition: "Words or phrases you select to describe what the work is about. These are used to search for content."
+   # usage_guidelines:
+    requirement: recommended #https://github.com/samvera/hyrax/issues/3298
+  #  controlled_values:
+  #    controlled_value_format:
+  #    controlled_value_sources:
+    sample_value: "afrofuturism"
+    property_uri: https://schema.org/keywords #https://github.com/samvera/hyrax/issues/1505
+    available_on_object:
+      - GenericWork
+      - Image
+    validations:
+      range: http://www.w3.org/2000/01/rdf-schema#Literal
+      data_type: http://www.w3.org/2001/XMLSchema#string
+  #    syntax:
+  #    validation_regex:
+      minimum_occurrence: 0
+  #    maximum_occurrence:
+    index_documentation: searchable
+  #  mapping:
+  #    service:
+  #    property_name:
+
+  - property_method_name: rights_statement
+    display_label: "Rights Statement"
+    definition: "Information about copyright, usage and access rights of the digital objects."
+   # usage_guidelines:
+    requirement: required
+    controlled_values:
+      controlled_value_format: http://www.w3.org/2001/XMLSchema#anyURI
+      controlled_value_sources:
+        - http://rightsstatements.org/
+        - http://creativecommons.org/
+    sample_value: http://rightsstatements.org/vocab/InC/1.0/
+    property_uri: http://www.europeana.eu/schemas/edm/rights
+    available_on_object:
+      - GenericWork
+      - Image
+    validations:
+      range: http://purl.org/dc/terms/RightsStatement
+      data_type: http://www.w3.org/2001/XMLSchema#anyURI
+  #    syntax:
+  #    validation_regex:
+      minimum_occurrence: 1
+      maximum_occurrence: 1
+    index_documentation: facetable
+    mapping:
+      service: "DPLA"
+      property_name: http://www.europeana.eu/schemas/edm/rights
+
+  - property_method_name: contributor
+    display_label: "Contributors"
+    definition: "A person or group you want to recognize for playing a role in the creation of the work, but not the primary role."
+  # usage_guidelines:
+    requirement: recommended
+  #  controlled_values:
+  #    controlled_value_format:
+  #    controlled_value_sources:
+    sample_value: "Harrison, Vashti"
+    property_uri: http://purl.org/dc/elements/1.1/contributor
+    available_on_object:
+      - GenericWork
+      - Image
+    validations:
+      range: http://www.w3.org/2000/01/rdf-schema#Literal
+      data_type: http://www.w3.org/2001/XMLSchema#string
+  #    syntax:
+  #    validation_regex:
+      minimum_occurrence: 0
+  #    maximum_occurrence:
+    index_documentation:
+      - searchable
+      - displayable
+    mapping:
+      service: "DPLA"
+      property_name: http://purl.org/dc/terms/contributor
+
+  - property_method_name: description
+    display_label: "Description" #https://github.com/samvera/hyrax/issues/3296 (this is Done in Hyrax 3.0.0)
+    # display_label: "Abstract or Summary" #Collection
```

Comment:
I had to comment out the duplicate keys.  Do we need a different way of providing different values in different contexts?